CMakeLists.txt

```diff
@@ -573,7 +573,10 @@ if (BUILD_TOOLS)
   )
 
   if(WITH_WASI)
+    set(CMAKE_POLICY_DEFAULT_CMP0169_BACK ${CMAKE_POLICY_DEFAULT_CMP0169})
```

Comment:
Can you add a comment here? Something like: "uvwasi uses the deprecated FetchContent_Populate function" ?